broker-daemon/block-order-worker/index.js

```diff
@@ -100,6 +100,41 @@ class BlockOrderWorker extends EventEmitter {
     }
 
     const blockOrder = new BlockOrder({ id, marketName, side, amount, price, timeInForce })
+    const baseEngine = this.engines.get(blockOrder.baseSymbol)
+    const counterEngine = this.engines.get(blockOrder.counterSymbol)
+    const { address: counterSymbolAddress } = await this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.counterSymbol})
+    const { address: baseSymbolAddress } = await this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.baseSymbol})
+
+    if (blockOrder.isBid) {
+      const outboundBalanceIsSufficient = await counterEngine.isBalanceSufficient(counterSymbolAddress, Big(blockOrder.counterAmount))
```

Comment:
does the `isBalanceSufficient` take `Big` as the type for it's amount? I don't think it should, and if it doesn't, I would not do this conversion here.